src/io/output_time.hh

```diff
@@ -194,8 +192,16 @@ public:
     template <typename T>
     ElementDataCache<T> & prepare_compute_data(std::string field_name, DiscreteSpace space_type, unsigned int n_rows, unsigned int n_cols);
 
-    /// Add information about field to map of used interpolations.
-    void add_field_interpolation(DiscreteSpace space_type, const std::string &field_name, unsigned int value_type);
+    inline Mesh * get_orig_mesh() {
```

Comment:
Check if we can remove orig mesh from OutputTime at all so it should work only with OutputMesh and data caches.